# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/font-matching/wingdings-1-ref.html
# Commit: cd5ac5f6ff6e
# Full Hash: cd5ac5f6ff6e626bd69e4d42ff9d1d7078abae03
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-03-01 09:40:29
# Regressor Bug: 1756720
# File Overlap Count: 1
# Description:
#   Bug 1756720 - When using legacy MS Symbol fonts, map character codes from the U+00xx range to the PUA range U+F0xx found in the cmap subtables. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D139724
# ==============================================================================

diff -r b547c1a2ef61 -r cd5ac5f6ff6e layout/reftests/font-matching/wingdings-1-ref.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/layout/reftests/font-matching/wingdings-1-ref.html	Mon Feb 28 20:38:38 2022 +0000
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<style>
+body {
+  font: 16px/2 sans-serif;
+}
+span {
+  font-family: Wingdings, sans-serif;
+}
+</style>
+<body>
+Trying the Wingdings font:
+<span>&#xf061;&#xf062;&#xf063;&#xf064;&#xf065;&#xf066;&#xf020;&#xf031;&#xf032;&#xf033;&#xf034;&#xf035;</span>